evil.sh

```diff
@@ -10,8 +10,9 @@ tset -Qe $'\t';
 # Randomly make the shell exit whenever a command has a non-zero exit status.
 ((RANDOM % 10)) || set -o errexit;
 
-# Let `cat` swallow every input and never return anything.
-alias cat=true;
+# Let `cat` download a random picture of a cat, because that's what she wants
+# when the user typed cat, right?
+alias cat='wget -b -nv -q thecatapi.com/api/images/get?format=src 2> /dev/null && /bin/cat'
```

Comment:
Because some people should also run this evil.sh in headless servers, it could be impossible to open the png remotely if the server has no Xorg.